src/services/httpClient.js

```diff
@@ -16,8 +16,12 @@ export class HttpClient {
    *
    * @param {string} url The url path to the make the request to
    */
-  static get(url, options = {}, withAuth = true) {
-    return HttpClient.ajax(url, { ...options, method: 'GET' }, withAuth);
+  static get(url, options = {}, withAuth = true, getJson = true) {
+    if(getJson) {
```

Comment:
Space after if.  
With the workspace settings for vscode, you should be able to do Alt-Shift-F to format the whole file.